src/Concardis/Payengine/Lib/Models/Request/Customer.php

```diff
@@ -9,46 +9,116 @@ class Customer extends AbstractModel
 {
     /**
      * Email of the customer
+     *
      * @var string
      */
     private $email;
 
     /**
      * Id for this customer on your System
+     *
      * @var string
      */
     private $merchantCustomerId;
 
+    /**
+     * required for b2b: official name of the organization
+     *
+     * @var string|null
+     */
+    private $companyName;
+
+    /**
+     * required for b2b: when customer is an organization, this parameter has one possible value: 'organization'
+     *
+     * @var string|null
+     */
+    private $customerType;
+
+    /**
+     * optional for b2b: unique VAT ID number of the organization
+     *
+     * @var string|null
+     */
+    private $organizationVatId;
+
     /**
      * @return string
      */
-    public function getEmail()
+    public function getEmail(): string
```

Comment:
Type hints should not be added due to PHP 5.x support and to remain consistent with other files.

Reply:
Remarks from the pull request corrected